hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteMergeOnRead.java

```diff
@@ -86,6 +88,12 @@ protected void checkWrittenData(File baseFile, Map<String, String> expected, int
     return EXPECTED1;
   }
 
+  @Disabled
+  @Test
+  public void testAppendOnly() throws Exception {
+    // Ignore the append only because only support cow.
```

Comment:
useless? or we can assert if we would throw an exception in MOR table?

Reply:
Yes, this is a better idea.